lib/workers/repository/onboarding/branch/onboarding-branch-cache.ts

```diff
@@ -80,6 +80,30 @@ export async function isOnboardingBranchModified(
   return isModified;
 }
 
+export function getOnboardingFileNameFromCache(): string | undefined {
+  const cache = getCache();
+  return cache.onboardingBranchCache?.configFileName;
+}
+
+export function getParsedOnboardingFileFromCache(): string | undefined {
+  const cache = getCache();
+  return cache.onboardingBranchCache?.configFileParsed;
+}
+
+export function setOnboardingConfigDetails(
+  configFileName: string,
+  configFileParsed: string
+): void {
+  const cache = getCache();
+  if (cache.onboardingBranchCache) {
+    const newOnboardingCache = structuredClone(cache.onboardingBranchCache);
```

Comment:
wonder why we need to clone here?

Reply:
I am skeptical of JS objects, so did it intuitively.